.gitignore

```diff
@@ -0,0 +1,3 @@
+extension.js
+node_modules
+package-lock.json
```

Comment:
the gitignore makes is so that when these files are generated locally, they don't get added to your repo. 

Generally, `package-lock.json` should be in your repo, but added it here since it wasn't in the repo previously